<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_Image">
      <property name="frameShape">
       <enum>QFrame::Shape::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Image"/>
   </widget>
   <widget class="QMenu" name="menuTransform">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Transform</string>
    </property>
    <addaction name="actionFlip_Vertical"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionGrayscale"/>
    <addaction name="actionProperties"/>
    <addaction name="actionProprietaryManipulation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImage"/>
   <addaction name="menuTransform"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="actionFlip_Vertical">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flip Vertical</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionProprietaryManipulation">
   <property name="text">
    <string>ProprietaryManipulation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
